addon/components/frost-link.js

```diff
@@ -71,6 +71,11 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
     icon: PropTypes.string,
     priority: PropTypes.oneOf(validPriorities),
     routeNames: PropTypes.array,
```

Comment:
What is the convention for deprecation? Should we only add a comment or is there anything else to do?